shared/serial_port_settings.py

```diff
@@ -190,11 +187,10 @@ def edit_page(self, tab: str):
         self.data_text = CTkLabel(self.edit_region, height=25, width=50, text="Waiting for Data...")
         self.data_text.grid(row=13, column=2, columnspan=3, padx=20, pady=3, sticky="ew")
 
-        self.serial_port_controller = SerialPortController()
-
-
-
+        self.back_to_summary_button = CTkButton(self.edit_region, text="Back to Summary", command=self.go_to_summary_page, height=14)
+        self.back_to_summary_button.grid(row=12, column=1, padx=20, pady=5, sticky="ns")
```

Comment:
This makes sense. When I was placing the button here, I couldn't get it to show up.